saleor/static/dashboard-next/siteSettings/components/SiteSettingsKeys/SiteSettingsKeys.tsx

```diff
@@ -0,0 +1,103 @@
+import Button from "@material-ui/core/Button";
+import Card from "@material-ui/core/Card";
+import IconButton from "@material-ui/core/IconButton";
+import { withStyles } from "@material-ui/core/styles";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import DeleteIcon from "@material-ui/icons/Delete";
+import * as React from "react";
+
+import CardTitle from "../../../components/CardTitle";
+import Skeleton from "../../../components/Skeleton";
+import i18n from "../../../i18n";
+import {
+  maybe,
+  renderCollection,
+  translatedAuthorizationKeyTypes
+} from "../../../misc";
+import { AuthorizationKeyType } from "../../../types/globalTypes";
+import { SiteSettings_shop_authorizationKeys } from "../../types/SiteSettings";
+
+interface SiteSettingsKeysProps {
+  disabled: boolean;
+  keys: SiteSettings_shop_authorizationKeys[];
+  onAdd: () => void;
+  onRemove: (name: AuthorizationKeyType) => void;
+}
+
+const decorate = withStyles(theme => ({
+  iconCell: {
+    "&:last-child": {
+      paddingRight: 0
+    },
+    width: 48 + theme.spacing.unit / 2
```

Comment:
Please name your magic numbers or, even better, extract them from the theme where available.

Reply:
And magic number here is `48`?